### ISSUE TITLE
Calling `..()` in a proc definition should be a compile error

### ISSUE THREAD

**Issue by ike709** [wixoaGit/OpenDream#575]
It should just be a compiler error instead.

**Comment by DamianX** [comment]
Can't reproduce this. Was it fixed?
I added these supercalls to TestGame @ 65c4ca98d61193d241b9362f35312903c1f143d5
```
	verb/tell_location()
		..()
		usr << "You are at ([x], [y], [z])"
```
```
/world/New()
	..()
	demo()
	world.log << "World loaded!"

/proc/demo()
	..()
```

**Comment by wixoaGit** [comment]
The run-time exception was fixed by the latest commit (65c4ca98d61193d241b9362f35312903c1f143d5). It is still not a compile-time error, which it would preferably be.
